modules/apps/josh-lapurd-theme/src/js/navbarFade.js

```diff
@@ -0,0 +1,27 @@
+window.onscroll
+
+function fadeNav(){  
+    var offset = getScrollXY();
+    offset[1] > 0 ? setNavOpacity(0.5) : setNavOpacity(0.0); 
```

Comment:
Write out ```if```/```else``` statements whenever possible.  We do not need an ```else``` statement when it is just an ```if```/```else```.  We can just do something like:

Ex.
```javascript
var opacity = 0.0;

if (offset[1] > 0) {
    opacity = 0.5;
}

setNavOpacity(opacity);
```